test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -276,7 +276,7 @@ public async Task GivenIncrementalLoad_MultipleImportsWithSameLastUpdatedAndImpl
             ndJson = CreateTestPatient(id, baseDate.AddYears(-1));
             location = (await ImportTestHelper.UploadFileAsync(ndJson, _fixture.StorageAccount)).location;
             request = CreateImportRequest(location, ImportMode.IncrementalLoad, false, true);
-            await ImportCheckAsync(request, null, 1);
+            await ImportCheckAsync(request, null, 0);
```

Comment:
So this is now considered a duplicate (and thus not an error)?

Reply:
The logic was faulty. Now when code sees input data that is identical to prior loaded (key is last updated only, as there is no version) it considers it already loaded, so no errors.